nautobot/extras/models/jobs.py

```diff
@@ -301,6 +291,22 @@ def job_task(self):
         except TypeError as err:  # keep 2.0-2.2.2 exception behavior
             raise NotRegistered from err
 
+    @property
+    def task_queues(self):
+        return self.job_queues.all().values_list("name", flat=True)
+
+    @task_queues.setter
+    def task_queues(self, value):
+        # value is going to be a comma separated list of queue names
+        # assume all queue types are celery for now
+        if isinstance(value, str):
+            value = value.split(",")
+        for queue in value:
+            job_queue, _ = JobQueue.objects.get_or_create(
+                name=queue, defaults={"queue_type": JobQueueTypeChoices.TYPE_CELERY}
+            )
```

Comment:
Do we need for backwards compatibility reasons to be able to support *creation* of JobQueue records through assignment to `task_queues = ...`? It feels a bit gross to me here, any chance we can make this require that any specified queues are already defined without it being a breaking change?

Reply:
It is for backwards-compatibility reasons. Require specified queues to be defined before setting `task_queues` as in raise an error if a specified queue does not exist?

Reply:
Raise a validation error if the task_queue specified does not exist.